commit 0d732a6cf7f7d27d42d7718dfe3fd89b14bea4de
Author: Kelly McLaughlin <kelly.mclaughlin@joyent.com>
Date:   2019-07-22T15:17:37-06:00 (3 months ago)
    
    MANTA-4426 electric-boray: service deployment should add dns name to SAPI application metadata

diff --git a/cmd/manta-init.js b/cmd/manta-init.js
index 06349cb..d6bd7cf 100755
--- a/cmd/manta-init.js
+++ b/cmd/manta-init.js
@@ -570,6 +570,8 @@ var pipelineFuncs = [
 		    extra.metadata['DOMAIN_NAME']);
 		extra.metadata['ELECTRIC_MORAY'] = sprintf('electric-moray.%s',
 		    extra.metadata['DOMAIN_NAME']);
+		extra.metadata['ELECTRIC_BORAY'] = sprintf('electric-boray.%s',
+		    extra.metadata['DOMAIN_NAME']);
 		extra.metadata['POSEIDON_UUID'] = POSEIDON.uuid;
 		extra.metadata['IMGAPI_SERVICE'] =
 		    url.format(self.IMGAPI.client.url);
